server/server.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
Since currently our services only use GET requests, I don't think we need to utilize `body-parser` yet since we aren't passing any data into the `req.body`.